docs/FAQ.md

```diff
@@ -56,7 +56,7 @@ No
 
 ### What is the policy on supporting end-of-life(EOL) Ruby versions?
 
-We will target the latest EOL Ruby version, please check the link below the see the current EOL version.
+We will target all non EOL Ruby versions, please check the link below the see the current EOL version.
```

Comment:
A perfect example of why I took the stance I did: https://github.com/sensu-plugins/sensu-plugins-resque/pull/5